testplan/web_ui/testing/src/AssertionPane/AssertionTypes/BasicAssertion.js

```diff
@@ -18,7 +18,7 @@ import {prepareBasicContent} from './basicAssertionUtils';
  * |~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~|
```

Comment:
This diagram could also be placed in the basicAssertionsUtils.js, but maybe it's redundant to have it twice

Reply:
I agree that having the diagram near the typedef would be useful! I think I might move it, no point in duplicating it.